dpnp/backend/kernels/dpnp_krnl_logic.cpp

```diff
@@ -71,6 +71,74 @@ void dpnp_all_c(const void* array1_in, void* result1, const size_t size)
     event.wait();
 }
 
+template <typename _DataType1, typename _DataType2, typename _ResultType>
+class dpnp_allclose_c_kernel;
+
+template <typename _DataType1, typename _DataType2, typename _ResultType>
+void dpnp_allclose_c(const void* array1_in, const void* array2_in, void* result1, const size_t size, double rtol, double atol)
```

Comment:
We already have `rtol` and `atol` in std namespace.
For better naming please rename them.